packages/@aws-cdk/aws-eks-next/lib/cluster.ts

```diff
@@ -80,30 +86,16 @@ export interface ICluster extends IResource, ec2.IConnectable {
 
   /**
    * Custom environment variables when running `kubectl` against this cluster.
-   * @default - no additional environment variables
    */
   readonly kubectlEnvironment?: { [key: string]: string };
 
-  /**
-   * A security group to use for `kubectl` execution.
-   *
-   * @default - If not specified, the k8s endpoint is expected to be accessible
-   * publicly.
-   */
-  readonly kubectlSecurityGroup?: ec2.ISecurityGroup;
-
   /**
    * Subnets to host the `kubectl` compute resources.
-   *
-   * @default - If not specified, the k8s endpoint is expected to be accessible
```

Comment:
I removed the `@default` annotations everywhere from `ICluster` and put them only where its being treated as a configuration (`ClusterAttributes/ClusterProps`)

Reply:
Makes sense